package.json

```diff
@@ -29,30 +29,32 @@
   },
   "homepage": "https://github.com/publiclab/Leaflet.DistortableImage",
   "dependencies": {
-    "grunt-cli": "^1.2.0"
-  },
-  "devDependencies": {
-    "chai": "^4.1.2",
     "exif-js": "^2.3.0",
     "font-awesome": "^4.2.0",
     "glfx": "0.0.4",
+    "jquery": "^3.4.0",
+    "leaflet": "^1.0.0",
+    "leaflet-toolbar": "^0.3.0",
+    "webgl-distort": "0.0.2"
+  },
+  "devDependencies": {
+    "chai": "^4.1.2",
     "grunt": "^1.0.3",
     "grunt-contrib-concat": "^1.0.1",
     "grunt-contrib-jshint": "^2.0.0",
     "grunt-contrib-watch": "^1.0.0",
     "grunt-karma": "^3.0.1",
-    "jquery": "~> 3.4.0",
     "karma": "^4.1.0",
     "karma-coverage": "^1.1.1",
     "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.5",
     "karma-phantomjs-launcher": "^1.0.4",
-    "leaflet": "^1.0.0",
-    "leaflet-toolbar": "^0.3.0",
     "matchdep": "^2.0.0",
     "mocha": "^6.0.0",
     "phantomjs-prebuilt": "^2.1.16",
-    "sinon": "^7.2.4",
-    "webgl-distort": "0.0.2"
+    "sinon": "^7.2.4"
+  },
+  "peerDependencies": {
```

Comment:
On the node.js site, there is an article on npm and in it is says that peerDependencies should be lenient: https://nodejs.org/es/blog/npm/peer-dependencies/ (scroll to bottom) 

I've never used peerDependencies myself, do you think this version requirement should be more lenient?

Would it be better to move all the dependency updates (`package.json` and `package-lock.json` into a new PR so it's easier to track?)

Reply:
Honestly i'm not that familiar with peerDependencies either... @rexagod any ideas?

Reply:
Ah! I remember running into an error concerning the CLI, since for some reason one of the packages (don't remember now, sorry!) `peer-depend`ed, i.e., only worked with a specific version of the cli for build purposes (could be karma but not sure). Therefore I put the cli in the peer dependencies to resolve such conflicts with the cli.

Reply:
@sashadev-sky You can put whatever version of a "middle-man" package (b/w a host and peer package) as long as it satisfies the requirements of the peer package. That way, npm knows exactly what version of the that package is the peer package compatible with.